packages/esm-form-engine-app/src/form-renderer/form-renderer.component.tsx

```diff
@@ -23,12 +27,24 @@ const FormRenderer: React.FC<FormRendererProps> = ({
   closeWorkspaceWithSavedChanges,
   encounterUuid,
   formUuid,
+  patient,
   patientUuid,
   promptBeforeClosing,
   visit,
 }) => {
   const { t } = useTranslation();
   const { schema, error, isLoading } = useFormSchema(formUuid);
+  const contentToPrintRef = useRef(null);
+  const onBeforeGetContentResolve = useRef<(() => void) | null>(null);
+  const [isPrinting, setIsPrinting] = useState(false);
+  const { printOptions } = useConfig<ConfigObject>();
+  const { excludePatientIdentifierCodeTypes } = useConfig<{
+    excludePatientIdentifierCodeTypes: {
+      uuids: string[];
+    };
+  }>({
+    externalModuleName: '@openmrs/esm-patient-banner-app',
+  });
```

Comment:
This ensures that we're properly fetching the `excludePatientIdentifierCodeTypes` config property from the Patient Banner app's config schema and annotating it appropriately. It defaults to `null` otherwise.